src/main/java/seedu/address/logic/parser/data/InstructionParser.java

```diff
@@ -16,12 +18,18 @@
      * @return ArrayList of String objects of the instructions in the parameter
      * @throws ParseException
      */
-    public static ArrayList<Instruction> parse(String instructionString) {
+    public static ArrayList<Instruction> parse(String instructionString) throws ParseException {
         String[] instructionsToken = instructionString.split("\\.");
         ArrayList<Instruction> instructions = new ArrayList<>();
         for (int i = 0; i < instructionsToken.length; i++) {
-            String instr = (i + 1) + ") " + instructionsToken[i].trim();
-            instructions.add(new Instruction(instr));
+            if (!instructionsToken[i].trim().equals("")) {
+                String instr = instructionsToken[i].trim();
+                instructions.add(new Instruction(instr));
+            }
+        }
+        if (instructions.size() == 0) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    "Instruction cannot be empty"));
```

Comment:
shouldnt the message of the parseException be like AddRecipeCommand.MESSAGE_EMPTY_INSTRUCTIONS